src/store/epics/accounts.ts

```diff
@@ -62,5 +63,5 @@ export const loadAccount$: RootEpic = (
 export const initiateLoadRequest$: RootEpic = (action$, state$) =>
   action$.pipe(
     filter(isActionOf([selectConnectedCurrency])),
-    map(() => accountLoadRequest(state$.value.wallets.activeWallet!.publicKey)),
+    map(() => accountLoadRequest(getActiveAccount(state$.value.wallet).keypair.publicKey())),
```

Comment:
I think this could be made a little more readable by splitting this into a separate mapping for each of the helper function calls, (and could use the withLatestFrom to be consistent with the other times we access `state$`) eg.

```typescript
  action$.pipe(
    filter(isActionOf(selectConnectedCurrency)),
    withLatestFrom(state$),
    map(([_, state]) => getActiveAccount(state.wallet)),
    map(wallet => accountLoadRequest(wallet.keypair.publicKey()))
```